plugins/org.jkiss.dbeaver.ext.databricks/plugin.xml

```diff
@@ -3,7 +3,8 @@
 <plugin>
 
     <extension point="org.jkiss.dbeaver.generic.meta">
-        <meta id="databricks" class="org.jkiss.dbeaver.ext.databricks.model.DatabricksMetaModel" driverClass="com.simba.spark.jdbc.Driver"/>
+        <meta id="databricks-simba" class="org.jkiss.dbeaver.ext.databricks.model.DatabricksMetaModel" driverClass="com.simba.spark.jdbc.Driver" dialect="spark"/>
+        <meta id="databricks-native" class="org.jkiss.dbeaver.ext.databricks.model.DatabricksMetaModel" driverClass="com.databricks.client.jdbc.Driver" dialect="spark"/>
```

Comment:
Please do it inline

https://github.com/dbeaver/dbeaver/blob/5463637ae1ad5764dc4ebaec4f3184c10da9faca/plugins/org.jkiss.dbeaver.ext.mssql/plugin.xml#L22-L25